### PR TITLE
Adjustment of sample code without

### PR DESCRIPTION
**Pull request by ericksposito**
We made an adjustment to the documentation because with the example we couldn't make it work.
Following the steps we put in the current example works perfectly. @padupe @monnylly @ericksposito @accezar

Signed-off-by: Erick Sposito <erick.sposito@madeiramadeira.com.br>

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/auth/identity-resolver.md [hunk 1]
```diff
@@ -273,46 +273,60 @@ that the user belongs to using the user access token in the provided result obje
...
-          kind: 'User',
-          name: localPart,
-          namespace: DEFAULT_NAMESPACE,
-        });
-
-        return ctx.issueToken({
-          claims: {
-            sub: userEntityRef,
-            ent: [userEntityRef],
+      ...defaultAuthProviderFactories,
```
Comment (benjdlambert) [inline docs/auth/identity-resolver.md:294]: Don't think that we particularly want to encourage this tbh. Should be enough without it.

File: docs/auth/identity-resolver.md [hunk 2]
```diff
@@ -273,46 +273,60 @@ that the user belongs to using the user access token in the provided result obje
...
+  DEFAULT_NAMESPACE,
+} from '@backstage/catalog-model';
 
 export default async function createPlugin(
-  // ...
-  return await createRouter({
-    // ...
+  env: PluginEnvironment,
+): Promise<Router> {
+  return createRouter({
```
Comment (benjdlambert) [inline docs/auth/identity-resolver.md:291]: Probably still want the `return await` here too.

**Review by ericksposito — COMMENTED** [review]
Adaptations according to the reviewer's suggestions

**Comment by benjdlambert** [comment]
Think you just need to run prettier on the file to make it look nice.

`$ yarn prettier --write docs/auth/identity-resolver.md` and commit the changes :()

**Comment by padupe** [comment]
Thanks @benjdlambert!
In the next PR's we will adopt all the recommended steps. :+1:

**Review by benjdlambert — APPROVED** [review]
Nice work! Thanks! :pray:

**Pull request merged**